api/pom.xml

```diff
@@ -124,10 +124,11 @@
 
 
 	<properties>
-		<annotation.api.version>2.1.0</annotation.api.version>
+		<annotation.api.version>3.0.0-M1</annotation.api.version>
 		<atinject.api.version>2.0.1</atinject.api.version>
 		<uel.api.version>5.0.0</uel.api.version>
 		<interceptor.api.version>2.1.0</interceptor.api.version>
```

Comment:
`2.2.0-RC1` here?

Or, if you have the powers, it would be great to have https://github.com/jakartaee/interceptors/pull/103 merged and another RC / milestone / whatever released.

Reply:
I went to do a release of interceptors and see there already is a 2.2.0-M1, so I guess you were talking about 2.2.0-RC1 being for interceptors. If this has been release we will have to stick with that, otherwise I'll redo it. Right now CDI does not build with that version so it does not appear to have made it to central, but it could be syncing.

Reply:
Thanks!

I was thinking we should bump the `interceptor.api.version` here, too. I don't know what would be the correct version string of the next Interceptors release (`2.2.0-M1` or `2.2.0-RC2` I guess), but I'm pretty sure we'll need that too.

Reply:
Ah yes, Interceptors has a release in the form of `2.2.0-RC1` quite a while ago. We can go with that, it just has slightly less precise specification text for the new methods, but the API is identical. Or we can do `2.2.0-RC2` (or `M1`?), I don't mind.

Reply:
I am trying to release a M1 right now. I am working on this and report back here once it is done.

Reply:
I kicked off a Jenkins build to create M1 a few minutes ago. However, the build does not like 2.2.0-M1, it is fine to take in 2.2.0.M1 instead. I got this error: 
```
WARNING: Bundle-Version (2.2.0.M1) should be 2.2.0-M1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  20.421 s
[INFO] Finished at: 2023-11-30T15:50:57Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.glassfish.build:spec-version-maven-plugin:2.1:check-module (default) on project jakarta.interceptor-api: spec verification failed. -> [Help 1]
Any thoughts on this @starksm64 ?
```

Reply:
See this PR, https://github.com/jakartaee/interceptors/pull/105

Reply:
Thanks @starksm64 ! I merged your PR and kicked off the Interceptor build 2.2.0-M1. It was done successfully. However, I could not push it out to maven central as I am unable to see the stage repo id from Sonatype. I need to request an access for that. Can you please release it to maven central and then you can update the dependency to that spec in CDI pom.xml?